atox/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,120 @@
+<resources>
+    <string name="accept">Akzeptieren</string>
+    <string name="add">Hinzufügen</string>
+    <string name="add_contact">Kontakt hinzufügen</string>
+    <string name="add_contact_message_error_empty">Nachricht darf nicht leer sein</string>
+    <string name="confirm">Bestätigen</string>
+    <string name="cancel">Abbrechen</string>
+    <string name="connecting">Verbindet…</string>
+    <string name="contact_typing">tippt…</string>
+    <string name="copied">Kopiert</string>
+    <string name="copy_atox">Kopieren</string> <!-- androidx.preference:preference has stolen copy -->
+    <string name="create">Erstellen</string>
+    <string name="create_a_tox_profile">Tox Profil erstellen</string>
+    <string name="delete_contact">Kontakt löschen</string>
+    <string name="message">Nachricht</string>
+    <string name="messages">Nachrichten</string>
+    <string name="messages_incoming">Eingehende Nachrichten</string>
+    <string name="name">Name</string>
+    <string name="name_default">aTox Benutzer</string>
+    <string name="no_contacts_call_to_action">Worauf wartest du? Füge einen Freund hinzu und starte sofort! </string>
+    <string name="navigation_drawer_close">Navigationsschublade schließen</string>
+    <string name="navigation_drawer_open">Navigationsschublade öffnen</string>
+    <string name="profile_photo_description">Profilbild</string>
+    <string name="reject">Ablehnen</string>
+    <string name="send">Senden</string>
+    <string name="sending">Wird gesendet…</string>
+    <string name="settings">Einstellungen</string>
+    <string name="status_indicator">Statusanzeige</string>
+    <string name="status_message">Status Nachricht</string>
+    <string name="tox_id">Tox-ID</string>
+    <string name="tox_id_error_length">Die Tox-ID ist immer 76 Zeichen lang (%1$d/76)</string>
+    <string name="tox_id_error_checksum">Prüfsumme ungültig</string>
+    <string name="tox_id_error_hex">Nur Hexadezimal erlaubt</string>
+    <string name="tox_id_share">Tox-ID teilen</string>
+    <string name="update">Aktualisieren</string>
+    <string name="quit">Beenden</string>
+    <string name="quit_confirm">Bist du sicher, dass du dich ausloggen willst?</string>
+    <!-- Friend request notification -->
+    <string name="friend_requests">Freundschaftsanfragen</string>
+    <string name="friend_request_from">Freundschaftsanfrage von %1$s</string>
+    <!-- Chat history clearing -->
+    <string name="clear_history">Verlauf löschen</string>
+    <string name="clear_history_cleared">Verlauf gelöscht</string>
+    <string name="clear_history_confirm">Bist du sicher, dass du den Verlauf mit %1$s löschen möchtest? </string>
```

Comment:
Trailing space at the end of the string. Is this intended?

Reply:
Also, "möchtest" and "willst" are synonyms. We have the latter twice, but the former is used once. I suggest changing to "willst" for consistency.